examples/draft-0-9-1/tex/js/components/TeXBlock.js

```diff
@@ -21,38 +21,26 @@ import {Entity} from 'draft-js';
 class KatexOutput extends React.Component {
   constructor(props) {
     super(props);
-    this._timer = null;
   }
 
   _update() {
-    if (this._timer) {
-      clearTimeout(this._timer);
-    }
-
-    this._timer = setTimeout(() => {
-      katex.render(
-        this.props.content,
-        this.container,
-        {displayMode: true},
-      );
-    }, 0);
+    katex.render(
+      this.props.content,
+      this.container,
+      {displayMode: true},
+    );
   }
 
   componentDidMount() {
     this._update();
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.content !== this.props.content) {
+  componentDidUpdate(prevProps) {
```

Comment:
Maybe worth calling out there's an unused `prevState` second argument here.

Reply:
Made the change but is that necessary?
The third `snapshot` argument is also unused.

And the docs for typical usage of componentDidUpdate don't include it either
https://reactjs.org/docs/react-component.html#componentdidupdate